draft_editor/draft_editor.py

```diff
@@ -0,0 +1,91 @@
+from talon import Module, actions, ui, app
+import time
+
+mod = Module()
+mod.mode("draft_editor", "Indicates whether the draft editor has been activated")
+
+default_names = [
+    "Visual Studio Code",
+    "Code",
+    "VSCodium",
+    "Codium",
+    "code-oss"
+]
+
+setting_editor_names = mod.setting(
+    "draft_editor",
+    type=str,
```

Comment:
Is there any reason not to use a list here for the setting itself, and set it to the default list? As it stands, the user would have to restart talon/touch this file for edits of the csv to take effect

Reply:
Settings do not support lists. Tried it but got an error so I changed to a csv instead.

Reply:
Oh, that's unfortunate. I believe settings do support dictionaries (e.g. dictate.word_map). While not ideal, maybe try that to preserve the reloading sweetness?

Reply:
I tried that as well but talon script only support string and numbers what I can tell. But that is not the only problem to get the reloading you want. I actually register an app at startup.
https://github.com/knausj85/knausj_talon/blob/d7f256fdd2725d89bb751109102c113e7a782c9b/draft_editor/draft_editor.py#L39

We would need to have a setting change event and dynamically update this app regular expression every time. For now I'm personally satisfied with having to restart when changing something as seldom as this. It is of course your call.

Reply:
I can live with it for now, thanks

Reply:
:)